src/rust/ide/src/model/module/plain.rs

```diff
@@ -35,8 +35,7 @@ impl Module {
     }
 
     fn notify(&self, notification:Notification) {
-        let notify  = self.notifications.publish(notification);
-        executor::global::spawn(notify);
+        self.notifications.notify(notification)
     }
```

Comment:
This function become so simple, that I think it could be "inlined" into current usages.